pkg/webhook/workspace/config.go

```diff
@@ -50,12 +45,6 @@ func Configure(ctx context.Context) error {
 		return err
 	}
 
-	// At this point we no longer need the cert generation deployment
-	err = deleteCertGenerationDeployment(ctx, c, namespace)
```

Comment:
If we don't remove cert generation deployment - we need to update the message here https://github.com/devfile/devworkspace-operator/blob/master/cert-generation/main.go#L124